__tests__/app.test.js

```diff
@@ -89,4 +102,27 @@ describe('app testing', () => {
             })
         })
     })
+    describe('post comments', () => {
+        test('201: add a comment and respond with the added comment', () => {
+            const newComment = {
+                username: "butter_bridge",
+                body: 'I love coding!'
+            }
+            return request(app)
+            .post('/api/articles/3/comments')
```

Comment:
This is lovely, but we also need a test to check that this has been posted to the database. Consider using your get comments endpoint after posting to assert that this new comment is now included.